### PR TITLE
AboutUs: move and expand "Powered by" line

### PR DESCRIPTION
**Pull request by pro-wh**
no issue filed, but I am to understand that the teams involved agreed to:

1. move the "Powered by ..." from the welcome and lock screens to the about screen
2. add Simply VC to this line

a screenshot of the new line

image

### CODE REVIEW

**Comment by peterjgilbert** [comment]
Can we make these links?

File: src/popup/pages/AboutUs/index.js [hunk 1]
```diff
@@ -83,6 +83,9 @@ class AboutUs extends React.Component {
           {this.renderTopInfo()}
           {this.renderWalletDetail()}
         </div>
+        <div className="about-tip-container">
+          <p className="about-tip">Powered by Bit Cat and Simply VC</p>
```
Comment (tjanez) [inline src/popup/pages/AboutUs/index.js:87]: Could you link these two to:
- https://www.bitcat365.com/en/
- https://simply-vc.com.mt/

**Review by tjanez — APPROVED** [review]
Otherwise, looks good:
Screenshot from 2021-09-08 23-05-11
Screenshot from 2021-09-08 23-08-20

File: src/popup/pages/AboutUs/index.js (hunk 1 above)
Comment (pro-wh) [reply src/popup/pages/AboutUs/index.js:87]: ok I'll try adding that to this pr

Comment (pro-wh) [reply src/popup/pages/AboutUs/index.js:87]: turns out it could use some design review. I'll put it in a separate PR.

**Pull request merged**